### ISSUE TITLE
Hellowallet entry has broken link

### ISSUE THREAD

**Issue by conorgil** [2factorauth/twofactorauth#3107]
The facebook link for the HelloWallet entry is broken. Leads to a 404 page over at Facebook.

I got this from the Travis link error log for proof external [here](https://travis-ci.org/2factorauth/twofactorauth/jobs/361973107).

In finance.yml:
```yaml
    - name: HelloWallet
      url: https://www.hellowallet.com
      twitter: HelloWallet
      facebook: hellowallet   <------ THIS IS WRONG. DUNNO WHAT IT SHOULD BE THO
      img: hellowallet.png
      tfa: No
```

**Comment by conorgil** [comment]
Related to #3103

**Comment by stephengroat** [comment]
closed, already addressed in #2921
